test/unit/metadata_gen.test.js

```diff
@@ -28,4 +25,23 @@ describe('MetadataGenerator', () => {
     const results = new MetadataGenerator(data).getMetadata();
     assert.strictEqual(expected, results);
   });
+
+  it('should handle cross-owner and cross-repo fixes properly', () => {
+    const { bodyHTML } = fixAndRefPR;
+    const newBodyHTML = bodyHTML.replace('#16437', '#123');
```

Comment:
This is not what that issue was asking for...it's more like testing that a PR with bodyHTML:

```
<p>Fixes: <a href="https://github.com/someone/node-core-utils/issues/16437" class="issue-link js-issue-link" data-error-text="Failed to load issue title" data-id="267969588" data-permission-text="Issue title is private" data-url="https://github.com/someone/node-core-utils/issues/16437">someone/node-core-utils#16437</a>
```

Would have a `Fixes: https://github.com/someone/node-core-utils/issues/16437` in the metadata